webapp-django/crashstats/signature/templates/signature/signature_report.html

```diff
@@ -5,8 +5,13 @@
     {{ super() }}
     {% compress css %}
 <link href="{{ static('crashstats/js/lib/select2/select2.css') }}" type="text/css" rel="stylesheet">
+<link href="{{ static('crashstats/css/lib/metricsgraphics.css') }}" type="text/css" rel="stylesheet">
     {% endcompress %}
 
+<link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.css') }}" media="screen" />
+<link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.structure.css') }}" media="screen" />
+<link rel="stylesheet" type="text/css" href="{{ static('crashstats/css/lib/jquery-ui.theme.css') }}" media="screen" />
```

Comment:
Are those still needed? I think you included them for the date input, but it's no longer there, so we can probably avoid included those css files, right?